proposals/deliverable-proposals/binding-templates.md

```diff
@@ -0,0 +1,28 @@
+## Bindings Templates
+
+In the next iteration of the binding templates, the group is planning the following work:
+
+1. Evaluation of going for the recommendation track or registry track and adopting one of them. The choice should allow:
+  1. Versioning of the documents in stable fashion that allow them to be used in implementations.
+  2. Documentation of the implementations of the individual binding templates.
+  3. Encourage new binding templates to be written by different (working group) members.
+  4. Publication of ontologies of protocols
+2. Publication of the currently existing binding templates (as registry or recommendation depending on the point above):
+  1. HTTP 
+  2. CoAP
+  3. MQTT
+  4. Modbus
+3. Starting work on the new binding template documents and publishing first working draft of each
+  1. Protocols
+    1. Bacnet
+    2. gRPC
+    3. OPC-UA
```

Comment:
here, we can also link to https://github.com/w3c/wot/blob/main/liaisons/opcf/tech_reqs.md